mantisshrimp/models/mantis_rcnn/mantis_faster_rcnn.py

```diff
@@ -4,16 +4,89 @@
 from mantisshrimp.core import *
 from mantisshrimp.models.mantis_rcnn.rcnn_param_groups import *
 from mantisshrimp.models.mantis_rcnn.mantis_rcnn import *
+from mantisshrimp.backbones import *
 
 
 class MantisFasterRCNN(MantisRCNN):
     @delegates(FasterRCNN.__init__)
-    def __init__(self, n_class, h=256, pretrained=True, metrics=None, **kwargs):
+    def __init__(
+        self,
+        n_class,
+        h=256,
+        backbone=None,
+        pretrained=True,
+        fpn=True,
+        metrics=None,
+        **kwargs,
+    ):
         super().__init__(metrics=metrics)
-        self.n_class, self.h, self.pretrained = n_class, h, pretrained
-        self.m = fasterrcnn_resnet50_fpn(pretrained=self.pretrained, **kwargs)
-        in_features = self.m.roi_heads.box_predictor.cls_score.in_features
-        self.m.roi_heads.box_predictor = FastRCNNPredictor(in_features, self.n_class)
+        self.n_class, self.h, self.backbone, self.pretrained, self.fpn = (
```

Comment:
Can we instead have one definition per line?

I also used to do it like this, but got convinced that doing one per line is a more readable way (we can see the formatter already breaks into multiple lines anyways, so no advantage in doing it like this)

Reply:
Reformatted, one definition per line now.